src/systems/filecoin_mining/sector/sealing.id

```diff
@@ -50,6 +50,19 @@ type SealCommitment struct {
     Expiration  block.ChainEpoch
 }
 
+type SectorPreCommitInfo struct {
+    SectorNumber
+    SealedCID       SealedSectorCID  // CommR
+    SealRandomness
```

Comment:
do we need `SealRandomness`? or should it be the `epoch` the randomness comes from?

Reply:
I think we can get the Randomness deterministically from the chain with epoch. Will change this to Epoch for now. cc @porcuquine

Reply:
Yes, I think the `Epoch` and `InteractiveEpoch` need to go on chain and are cheaper there than the randomness itself, which can be looked up again when needed. Once looked up for verification, the randomness itself can be passed to `VerifySeal`.